### PR TITLE
Feature: zookeeper upgrade path

### PR DESCRIPTION
**Pull request by avnik**
Folks, please check both  clean install and update from 0.5.0 tag.

Another issue still be, zk logs no more available to logstash, IDK if it really need (it not debug logs, it transaction logs -- db of zk)

### CODE REVIEW

File: roles/zookeeper/files/zookeeper-update-mantl-0.5-0.6.sh [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+ZKUSER=${1:-zookeeper}
+
+if test -d /var/lib/zookeeper/version-2; then
+    # Already upgraded, or fresh 0.6.x setup
+    exit 0
+fi
+
+if systemctl show zookeeper | grep -q docker >/dev/null; then
+    systemctl stop zookeeper
+    rm -f /usr/lib/systemd/system/zookeeper.service
```
Comment (stevendborrelli) [inline roles/zookeeper/files/zookeeper-update-mantl-0.5-0.6.sh]: Do we have to run `systemd disable/enable` here?

File: roles/zookeeper/files/zookeeper-update-mantl-0.5-0.6.sh [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    systemctl daemon-reload
+fi
+
+if test -d /var/log/zookeeper/version-2; then
+    mkdir -p /var/lib/zookeeper
+    mv /var/log/zookeeper/version-2 /var/lib/zookeeper/version-2
+    chown -R zookeeper /var/lib/zookeeper/version-2
```
Comment (stevendborrelli) [inline roles/zookeeper/files/zookeeper-update-mantl-0.5-0.6.sh]: should this be $ZKUSER?

**Comment by avnik** [comment]
Fixes mentioned by @stevendborrelli added,  checked myself on vagrant and openstack. Could someone other check it and merge?

**Comment by ryane** [comment]
+1 for a clean install on gce

**Comment by ryane** [comment]
an upgrade from 0.5 went smoothly on AWS

**Pull request merged**